src/pages/docs/test-step-recorder/install-chrome-extension.md

```diff
@@ -25,7 +25,7 @@ With Testsigma's Chrome extension, users can quickly record test steps by captur
 
 ## **Steps to Install Chrome Extension**
 
-1. Go to [Testsigma Recorder](https://chrome.google.com/webstore/detail/testsigma-recorder/epmomlhdjfgdobefcpocockpjihaabdp)
+1. Go to <a href="https://chrome.google.com/webstore/detail/testsigma-recorder/epmomlhdjfgdobefcpocockpjihaabdp" target="_blank">Testsigma Recorder</a>
```

Comment:
@bharathk08  Why do we need to embed HTML code directly? Is the purpose for adding the target attribute in a link?

Reply:
The link is leading in the same tab bro. I was told to open that link in new tab. So, made changes.

Reply:
Yeah, that's what I'm referring.
I'll share you the syntax for this. So that, there is no HTML tag need for here.